### ISSUE TITLE
`--rwdir=guestpath=.` option causes crash

### ISSUE THREAD

**Issue by musamaanjum** [arighi/virtme-ng#117]
I'm using the Arm64 Jammy release. I'm trying to share a directory on amd64 with the arm64 guest by using `--rwdir=guestpath=.` option as --rwdir=/root/guestpath=/abc. There are some permissions denied and some other errors. 

```
# works
vng --arch arm64 --cpus 8 --user root --root $rootfs/ubuntu22

# sharing doesn't work
vng --arch arm64 --cpus 8 --user root --root $rootfs/ubuntu22 --rwdir=/root/guestpath=/abc
mkdir: cannot create directory '/root/guestpath': Permission denied
mount: /root/guestpath: cannot mount virtme.initmount0 read-only.
# kernel panics
Kernel panic - not syncing: Attempted to kill init! exitcode=0x00000100
```
`--rwdir` options adds the following to qemu command line:
```
-fsdev local,id=virtfs13,path=.,security_model=none,multidevs=remap -device virtio-9p-device,fsdev=virtfs13,mount_tag=virtme.initmount0 virtme_initmount0=/root/guestpath
```

I think `--rw` is unrelated to above dierctory sharing. But if we add `--rw` option:
```
# adding --rw causes more errors
vng --arch arm64 --cpus 8 --user root --root $rootfs/ubuntu22-arm-2 --rw --rwdir=/root/guestpath=/abc
touch: cannot touch '/run/tmp/fstab': No such file or directory
mount: /etc/fstab: special device /run/tmp/fstab does not exist.
cp: cannot create regular file '/run/tmp/hosts': No such file or directory
...
/run/virtme/guesttools/virtme-init: line 123: /run/tmp/shadow: No such file or directory
mkdir: cannot create directory '/root/guestpath': Permission denied
mount: /etc/shadow: special device /run/tmp/shadow does not exist.
mount: /root/guestpath: cannot mount virtme.initmount0 read-only.
```

**Comment by arighi** [comment]
This happens because the guest (that runs as regular user) needs to be able to create the guestpath mount point inside the guest.

vng creates some overlays that allow to get ephemeral write access to certain dirs (see https://github.com/arighi/virtme-ng/blob/main/virtme_ng/run.py#L801), so basically the guestpath mount-point needs to be created inside one of these dirs.

Can you try to create the guestpath inside /srv for example (`--rwdir=/srv/guestpath=/abc`), assuming it exists in your chroot?

**Comment by arighi** [comment]
This is addressed in #199. Closing for now.

**Comment by musamaanjum** [comment]
Thank you so much for addressing it in https://github.com/arighi/virtme-ng/pull/119. I'll try and test it.
